src/main/scala/com/github/sadikovi/spark/rdd/NetFlowFileRDD.scala

```diff
@@ -121,24 +144,36 @@ private[spark] class NetFlowFileRDD[T<:SQLRow: ClassTag] (
         reader.prepareRecordBuffer(internalColumns)
       }
 
-      val rawIterator = recordBuffer.iterator().asScala
+      val rawIterator: java.util.Iterator[Array[Object]] = recordBuffer.iterator()
```

Comment:
I do not think we need explicit import, either add shortcut import or remove type at all.